bundles/CoreBundle/Migrations/Version20220120121803.php

```diff
@@ -32,111 +32,56 @@ public function up(Schema $schema): void
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_link`
-            ADD CONSTRAINT `fk_documents_link_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_page`
-            ADD CONSTRAINT `fk_documents_page_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_snippet`
-            ADD CONSTRAINT `fk_documents_snippet_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_printpage`
-            ADD CONSTRAINT `fk_documents_printpage_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_email`
-            ADD CONSTRAINT `fk_documents_email_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `email_log`
-            CHANGE `documentId` `documentId` int(11) unsigned NULL,
-            ADD CONSTRAINT `fk_email_log_documents`
-            FOREIGN KEY (`documentId`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_newsletter`
-            ADD CONSTRAINT `fk_documents_newsletter_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_editables`
-            ADD CONSTRAINT `fk_documents_editables_documents`
-            FOREIGN KEY (`documentId`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `documents_translations`
-            ADD CONSTRAINT `fk_documents_translations_documents`
-            FOREIGN KEY (`id`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
-
-        $this->addSql('ALTER TABLE `sites`
-            ADD CONSTRAINT `fk_sites_documents`
-            FOREIGN KEY (`rootId`)
-            REFERENCES `documents` (`id`)
-            ON UPDATE NO ACTION
-            ON DELETE CASCADE;');
+        foreach(['documents_hardlink', 'documents_link', 'documents_page', 'documents_snippet', 'documents_printpage', 'documents_email', 'documents_newsletter', 'documents_editables', 'documents_translations'] as $table) {
```

Comment:
```suggestion
        foreach(['documents_hardlink', 'documents_link', 'documents_page', 'documents_snippet', 'documents_printpage', 'documents_email', 'documents_newsletter', 'documents_translations'] as $table) {
```

for `documents_editables` is [different](https://github.com/pimcore/pimcore/pull/14199/files?w=1#diff-16a66ecef029997901ca218419d1d956f02bad7ae4361808f4eee946aba41134L93-L94)
```
            FOREIGN KEY (`documentId`)
            REFERENCES `documents` (`id`)
```
and not `FOREIGN KEY ('id')`

Reply:
Fixed. Now this gets handled separately.